styles.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+html {
+  font-family: sans-serif, 'Poppinsp';
+}
+
+body {
+  background-color: #e5e5e5;
+  height: 918px;
+  width: 375px;
+  margin: 0;
+}
+
+header {
+  background-color: white;
+}
+
+main {
+  background-image: url("images/header-shapes\ mobile.svg"), url("images/header-background-mobile\(4\).svg");
+  height: 918px;
+  width: 375px;
+  background-repeat: no-repeat;
+  padding-top: 120px;
+}
+
+.navigation {
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  text-decoration: none;
+  margin-top: 58px;
+  margin-left: 24px;
+  width: 76px;
+  height: 20px;
+  font-family: sans-serif, 'Poppinsp';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+.humburger {
+  margin-top: 68px;
+  margin-right: 36px;
+}
+
+.logo:hover {
+  text-decoration: underline;
+}
+
+.hero {
+  margin: 0 24px;
+  width: 327px;
+  height: 150px;
+  font-family: sans-serif, 'Poppinsp';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+}
+
+.hero-details {
+  margin: 12px 24px 0 24px;
+  width: 327px;
+  height: 146px;
+  font-family: sans-serif, 'Poppinsp';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.connect {
+  width: 127px;
+  height: 24px;
+  font-family: sans-serif, 'Poppinsp';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 234px 0 24px;
+}
+
+.link {
+  display: flex;
+  list-style: none;
+  gap: 10px;
```

Comment:
- [ ] Kindly consider changing this gap from `10px` to `19px` as being shown in the Figma design because we need to stick to the figma design.